packages/block-editor/src/filters/with-emphasis-control/controls.js

```diff
@@ -0,0 +1,27 @@
+import { __ } from "@wordpress/i18n";
```

Comment:
This control is strictly related to the **Supernova** block. It'd be better if we keep it in the block's source folder as there could be conditions of displaying the control only related to that block and I don't really see any other block using it (besides Media Card block which would eventually become deprecated, and having another version of this control until it's eventually removed doesn't necessarily seem like a bad idea for the time being)

Reply:
If we were to treat it like a general control that can have different implications for other blocks, the best place for it would've remained the color signal package. However having SlotFills in our ControlsSections that can be used even by third party blocks seems like the better way to go forward.

Reply:
After a short discussion, we have decided that for the moment it's better if this control will remain in the current folder.